packages/serverpod/lib/src/web_server/web_widget.dart

```diff
@@ -42,13 +42,13 @@ class Component extends AbstractComponent {
   }
 }
 
-/// Combines a List of [Component]s into a single component.
-class ListComponent extends AbstractComponent {
+/// Combines a List of widgets into a single widget.
```

Comment:
Use this when we can.

```suggestion
/// Combines a List of [WebWidget]s into a single [WebWidget].
```